### ISSUE TITLE
Some suggestions for improving the Gallery Slider widget - jet product gallery

### ISSUE THREAD

**Issue by 79ho3ein** [CrocoBlock/suggestions#7033]
Please consider these few things to improve this widget as they are very important.

1- padding for product tumbnails.

2- padding for images : https://prnt.sc/Gc3LORWNxpbt

3- Gap option to control the distance between the featured image and thumbnails : https://prnt.sc/WQkBwavCkxUR

4- Left and right position for thumbnails : https://prnt.sc/10fimlfbWzE0

For example, if we choose the left position, the result will be like this: https://prnt.sc/lEvMa5uKlos-

5- box shadow for Prev/Next Arrows (featured image and thumbnails)

6- inside and outside position for Prev/Next Arrows (Like Elementor's slider widget)
This option is very useful and helps to keep the Prev/Next Arrows in the widget frame.

7- speed slide control for thumbnails

thank you

**Comment by olyaastep** [comment]
Add an ability to create gallery like on airbnb : https://www.airbnb.com/rooms/48821490?adults=1&category_tag=Tag%3A7769&children=0&enable_m3_private_room=true&infants=0&pets=0&photo_id=1150952775&search_mode=flex_destinations_search&check_in=2023-09-01&check_out=2023-09-08&source_impression_id=p3_1692639217_%2B1jmoy5KL9PKr4%2FL&previous_page_section_name=1000&federated_search_id=bded5b55-573b-4064-849a-b3e795a7c3ba

**Comment by k4znIm** [comment]
I agree. I use that widget a lot, but its kinda limited, what you can do. 

I would also add:

- object-fit option for main image and thumbnails
- In vertical menu possibility to move tumbnails images inside slider into middle, top or bottom. Now they are on top and you can't adjust it.
- Possiblitity to change thumbnails width and height.
- Remade it with flexboxes for better customization

There are some css i use for horizontal gallery to achieve some of these.

Custom ID is "gallery"

```
/* ==== Jet Slider Gallery ==== */

/* Change main image height and object-fit */
#gallery .jet-woo-product-gallery__image-link img{
    height: 450px;
    object-fit: contain;
}

/* Change thumbnails images height and object-fit */
#gallery .jet-woo-swiper-control-thumbs__item-image img{
    height: 120px;
    object-fit: contain;
}

/* Change thumbnails container padding (gap between main image and thumbnails) */
.jet-gallery-swiper-thumb{
    padding: 0px 10px 0px 10px !important;
}

/* Same settings just for phone */
@media only screen and (max-width: 767px) {
  #gallery .jet-woo-product-gallery__image-link img{
        height: 350px;
        object-fit: contain;
	}

	#gallery .jet-woo-swiper-control-thumbs__item-image img{
    height: 100px;
		object-fit: contain;
	}
	
	.jet-gallery-swiper-thumb{
		padding: 0px 10px 0px 10px !important;
	}
}
```

**Comment by k4znIm** [comment]
> Add an ability to create gallery like on airbnb : https://www.airbnb.com/rooms/48821490?adults=1&category_tag=Tag%3A7769&children=0&enable_m3_private_room=true&infants=0&pets=0&photo_id=1150952775&search_mode=flex_destinations_search&check_in=2023-09-01&check_out=2023-09-08&source_impression_id=p3_1692639217_%2B1jmoy5KL9PKr4%2FL&previous_page_section_name=1000&federated_search_id=bded5b55-573b-4064-849a-b3e795a7c3ba

That's would be great, but rather, than make it possible in slider widget, that would be new widget completly.

**Comment by ArongodCoder** [comment]
@79ho3ein 

Thank you for your detailed feedback and suggestions regarding the widget improvements. We have carefully reviewed your request and outlined the following points and implementation considerations based on your input:

1. You can control the spacing between thumbnails via Tab Content → Slider → [Space Between](https://prnt.sc/OAcRN2_eLpnr) and Tab Style → Thumbnails → [Gutter](https://prnt.sc/I4r-NFGFG8ZE). These options effectively replicate the effect of padding.
2. [Padding for Images](https://prnt.sc/oGRDAJinsAyC) has been added under Tab Style → Images, allowing more control over image spacing within the gallery.
3. The gap between the featured image and thumbnails can be adjusted using Tab Style → Thumbnails → [Gutter](https://prnt.sc/I4r-NFGFG8ZE).
4. To set the Left or Right position for thumbnails, switch to Tab Content → Slider → [Direction: Vertical](https://prnt.sc/Fo___LhKl3mU). Once enabled, additional [alignment settings](https://prnt.sc/JqjEV8zrDQAT) become available.
5. Currently, Elementor does not provide native support for applying box shadows to SVG elements such as the navigation arrows. We will monitor future updates and consider implementing this feature once support becomes available.
6. The positioning of Prev/Next Arrows (inside or outside) can be adjusted via Tab Style → Images → [Indent](https://prnt.sc/EhJ9UTeZz23W), with a similar control also available for thumbnails.
7. Thank you for your suggestion regarding the speed control for thumbnail slides. At this time, we do not have plans to implement this feature. However, we appreciate your feedback and will keep it in mind for future improvements.

**Comment by ArongodCoder** [comment]
@olyaastep 
Thank you for your suggestion! We're currently working on adding a gallery layout similar to Airbnb. This feature is planned and will be available in one of the upcoming updates.

**Comment by ArongodCoder** [comment]
@k4znIm
Thank you for sharing your feedback and suggestions. At this time, we do not plan to add these specific options to the widget. However, you can still implement these customizations on your own using CSS or by extending the widget via available hooks and template overrides in your project.

**Comment by aalexandr21** [comment]
was fixed and released in JetGallery 2.2.0
